root/scripts/vscripts/community/functions.nut

```diff
@@ -174,6 +174,43 @@ function make_navblock ( user_strTargetname,
 	}
 }
 
+function make_trig_godspot(area_origins) {
+	local areas = []
+	foreach(origin in area_origins) {
+		local area = NavMesh.GetNearestNavArea(origin, 16, true, true);
+		if(area == null) {
+			printl("godspot couldn't be created because area at " + origin + " wasn't found")
+			return
+		}
+		areas.append(area)
+	}
+	local godspot = areas[0];
+	local disconnect = areas.slice(1)
```

Comment:
Errors if `area_origins` has 0 elements, might be worth checking and returning?

Reply:
error's only if someone uses the function with wrong parameter so I don't think it's worth checking

Reply:
Newest edit ensures that areas have at least 1 element when it gets to this point